### PR TITLE
Updated kubernetes version to latest

### PR DESCRIPTION
**Pull request by shreekarSS**
setup.py: Updated kubernetes version to latest

Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

**Comment by shreekarSS** [comment]
ocs-ci) [sshreeka@localhost ocs-ci]$ pip show kubernetes
Name: kubernetes
Version: 9.0.0
Summary: Kubernetes python client
Home-page: https://github.com/kubernetes-client/python
Author: Kubernetes
Author-email: 
License: Apache License Version 2.0
Location: /home/sshreeka/wip/ocs/test_setup/ocs-ci/lib/python3.7/site-packages
Requires: urllib3, setuptools, requests-oauthlib, six, requests, websocket-client, certifi, python-dateutil, pyyaml, google-auth

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
         'paramiko==2.4.2',
         'pyyaml>=4.2b1',
         'jinja2==2.10.1',
-        'kubernetes<9.0.0,>=8.0.0',  # to satisfy openshift package requirement
+        'kubernetes>=9.0.0',  # to satisfy openshift package requirement
```
Comment (clacroix12) [inline setup.py]: I think the comment here was to justify why we specified <9>=8. As long as this has been tested and works with 9.0.0 let's remove the comment as well.

Comment (shreekarSS) [reply setup.py]: Yep, will remove

Comment (clacroix12) [reply setup.py]: @shreekarSS please update this PR so we can approve and merge, thanks!

**Comment by vasukulkarni** [comment]
@shreekarSS mind testing this one more time please ? Thanks

**Comment by shreekarSS** [comment]
> @shreekarSS mind testing this one more time please ? Thanks

Yes @vasukulkarni , will raise new pr for the same

**Pull request closed without merging**